dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx

```diff
@@ -84,6 +85,28 @@ cpdef dparray dpnp_diag(v, k):
     return result
 
 
+cpdef dparray dpnp_full(result_shape, value_in, result_dtype):
+    """ Convert string type names (dparray.dtype) to C enum DPNPFuncType """
```

Comment:
```suggestion
    # Convert string type names (dparray.dtype) to C enum DPNPFuncType
```
Please use comment sign for comments otherwise Cython converts this to strings from src.